<launch>
  <node pkg="colibri_laser" name="lms1xx_cartogmap" type="LMS1xx_cartogmap" output="screen">
  </node>
</launch>
